transform/models/intermediate/usace/int_usace__most_recent_debris_removal.sql

```diff
@@ -0,0 +1,30 @@
+with pdr as (
+
+    select * from {{ ref('stg_usace__parcel_debris_removal') }}
+
+),
+
+most_recent_date as (
+    select MAX(last_updated) as max_date
+    from pdr
+),
+
+
+most_recent_pdr_data as (
+
+    select 
+        epa_status,
+        roe_status,
+        hsa_status,
+        pcr_status,
+        fso_pkg_returned,
+        object_id,
+        _load_date,
+        last_updated
+    
+    from pdr
+    inner join most_recent_date as md on pdr.last_updated = md.max_date
```

Comment:
```suggestion
    where pdr.last_updated = (select max(last_updated) from pdr)
```

Not a big deal for tables this small, but a filter should be faster and have less room for accidental cardinality changes than a join.

Reply:
Very true thank you!